### ISSUE TITLE
App Crash: While creating new policy app is crashing

### ISSUE THREAD

**Issue by anildahalsussol** [openmsupply/mobile#5043]
## Describe the bug

While creating new policies while dispensing app is crashing.

### To reproduce

Steps to reproduce the behaviour:

1. Create prescriptions for a patient with items.
2. Create a new policy.
4. Type the text in family policy and enter.

### Expected behaviour

App should not crash.

### Proposed Solution

Leave if you don't know how to fix/implement. Edit this issue description and explain here if you know the best path of implementing the fix within the codebase.

### Version and device info

- App version: 8.6 rc03
- Tablet model: lenovo
- OS version:

**Comment by sworup** [comment]
```
Warning: Failed prop type: ForwardRef(_c): prop type `isDisabled` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.This often happens because of typos such as `PropTypes.function` instead of `PropTypes.func`.
_c@http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=com.msupplymobile&modulesOnly=false&runModule=true:366731:21
    in FormControlComponent (created by Connect(FormControlComponent))
    in Connect(FormControlComponent) (at DispensingPage.js:259)
    in RCTView (at View.js:34)
    in View (at ModalContainer.js:89)
    in RCTView (at View.js:34)
    in View (at ModalContainer.js:87)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:107)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:134)
    in AppContainer (at Modal.js:228)
    in RCTView (at View.js:34)
    in View (at Modal.js:249)
    in VirtualizedListContextResetter (at Modal.js:247)
    in RCTModalHostView (at Modal.js:234)
    in Modal (at ModalContainer.js:80)
    in RCTView (at View.js:34)
    in View (at ModalContainer.js:79)
    in ModalContainer (at DispensingPage.js:254)
    in Dispensing (created by Connect(Dispensing))
    in Connect(Dispensing) (at SceneView.tsx:122)
    in StaticContainer
    in EnsureSingleNavigator (at SceneView.tsx:114)
    in SceneView (at useDescriptors.tsx:153)
    in RCTView (at View.js:34)
    in View (at CardContainer.tsx:245)
    in RCTView (at View.js:34)
    in View (at CardContainer.tsx:244)
    in RCTView (at View.js:34)
    in View (at CardSheet.tsx:33)
    in CardSheet (at Card.tsx:573)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:217)
    in AnimatedComponent (at createAnimatedComponent.js:278)
    in AnimatedComponentWrapper (at Card.tsx:555)
    in PanGestureHandler (at GestureHandlerNative.tsx:13)
    in PanGestureHandler (at Card.tsx:549)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:217)
    in AnimatedComponent (at createAnimatedComponent.js:278)
    in AnimatedComponentWrapper (at Card.tsx:544)
    in RCTView (at View.js:34)
    in View (at Card.tsx:538)
    in Card (at CardContainer.tsx:206)
    in CardContainer (at CardStack.tsx:623)
    in RNSScreen (at createAnimatedComponent.js:217)
    in AnimatedComponent (at createAnimatedComponent.js:278)
    in AnimatedComponentWrapper (at src/index.native.tsx:127)
    in Screen (at Screens.tsx:74)
    in MaybeScreen (at CardStack.tsx:616)
    in RNSScreenContainer (at src/index.native.tsx:165)
    in ScreenContainer (at Screens.tsx:50)
    in MaybeScreenContainer (at CardStack.tsx:498)
    in CardStack (at StackView.tsx:462)
    in KeyboardManager (at StackView.tsx:458)
    in RNCSafeAreaProvider (at SafeAreaContext.tsx:76)
    in SafeAreaProvider (at SafeAreaProviderCompat.tsx:42)
    in SafeAreaProviderCompat (at StackView.tsx:455)
    in GestureHandlerRootView (at GestureHandlerRootView.android.tsx:26)
    in GestureHandlerRootView (at StackView.tsx:454)
    in StackView (at createStackNavigator.tsx:87)
    in StackNavigator (at mSupplyMobileApp.js:255)
    in RCTView (at View.js:34)
    in View (at mSupplyMobileApp.js:254)
    in MSupplyMobileAppContainer (created by Connect(MSupplyMobileAppContainer))
    in Connect(MSupplyMobileAppContainer) (at main.js:31)
    in EnsureSingleNavigator (at BaseNavigationContainer.tsx:409)
    in BaseNavigationContainer (at NavigationContainer.tsx:91)
    in ThemeProvider (at NavigationContainer.tsx:90)
    in NavigationContainer (at main.js:30)
    in Provider (at main.js:29)
    in ErrorHandler (at main.js:28)
    in App (at renderApplication.js:47)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:107)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:134)
    in AppContainer (at renderApplication.js:40)
```

**Comment by sworup** [comment]
Resolved by #5058

**Comment by santoshkhanal12** [comment]
Tested on mobile version: 8.6.2.rc-04

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] In dispensing click on dispense, select a prescriber, select an item, give quantity and click next. 
- [ ] In the payment section click the plus button to add an insurance policy
- [ ] Previously it was crashing now it should not. **-PASS-**
      @sworup the mentioned issue is fixed. But found out app crashing bug inside the Insurance policy window. So considering this issue as done. And created new issue here: https://github.com/openmsupply/mobile/issues/5080
